### DISCUSSION TITLE
docs: SchemaHelper is not exported from core rather than knex?

### DISCUSSION THREAD

**Discussion by Clex1o1** [mikro-orm/mikro-orm#4328] [General]
Hi there! In the documentation I found the section about custom driver. It tells me to implement a Schema class which extended the SchemaHelper class: https://mikro-orm.io/docs/custom-driver#schemahelper
```
import { SchemaHelper } from '@mikro-orm/core';

export class MyCustomSchemaHelper extends SchemaHelper {
```

SchemeHelper is not exported from @mikro-orm/core but I found it in the knex package. Should the documentation be corrected to install the knex package and import the class from it? Or did I miss something im my setup?

**Comment by B4nan** [comment]
Depends on what driver you want to build, if its nosql driver, there is no schema and therefore no schema helper. The concept of `SchemaHelper` is rather optional. Will update the docs a bit now, planning to do a bigger overhaul before v6 gets out.

**Reply by Clex1o1** [reply]
Thanks for your super quick reply! 
I was starting playing around with possibles of a custom driver. I'd like to test it with JSON and on top of https://unstorage.unjs.io.

**Reply by B4nan** [reply]
Take a look at how the mongo driver works.

**Reply by Clex1o1** [reply]
I will thanks a lot.
